crates/evm/evm/src/executors/invariant/error.rs

```diff
@@ -2,17 +2,24 @@ use super::{BasicTxDetails, InvariantContract};
 use crate::executors::{Executor, RawCallResult};
 use alloy_json_abi::Function;
 use alloy_primitives::{Address, Bytes};
-use ethers_core::types::Log;
+use ethers_core::{
+    rand::{seq, thread_rng, Rng},
+    types::Log,
+};
 use eyre::Result;
 use foundry_common::contracts::{ContractsByAddress, ContractsByArtifact};
 use foundry_evm_core::{constants::CALLER, decode::decode_revert};
 use foundry_evm_fuzz::{BaseCounterExample, CounterExample, FuzzedCases, Reason};
 use foundry_evm_traces::{load_contracts, CallTraceArena, TraceKind, Traces};
+use itertools::Itertools;
 use parking_lot::RwLock;
 use proptest::test_runner::TestError;
+use rayon::iter::{IntoParallelRefIterator, ParallelIterator};
 use revm::primitives::U256;
 use std::sync::Arc;
 
+type ShrunkCallIndices = Vec<usize>;
```

Comment:
not a big fan of type aliases—I think it's fine to just use `Vec<usize>` as long as everything has a clear name